charts/topolvm/Chart.yaml

```diff
@@ -4,7 +4,7 @@ type: application
 home: https://github.com/topolvm/topolvm
 name: topolvm
 description: Topolvm
-version: 15.5.2
+version: 15.6.0
```

Comment:
You don't have to update the version in this PR. We adopt a monthly release cycle.